mlflow/utils/__init__.py

```diff
@@ -63,6 +63,11 @@ def reraise(tp, value, tb=None):
         tb = None
 
 
+def chunk_list(l, chunk_size):
```

Comment:
Define a utility to chunk a list, which is used to create GetCredentialsForRead and GetCredentialsForWrite requests that respect the maximum request size of 2000 artifact paths.